src/components/GoogleOptions/index.js

```diff
@@ -0,0 +1,66 @@
+import { Tooltip } from 'antd';
+import { AiOutlinePlus} from 'react-icons/ai';
+import { FcManager, FcCalendar, FcCheckmark, FcIdea } from 'react-icons/fc';
+import { urls } from '../../constant/urls';
+import { Container } from "./styles";
+
+const GoogleOptions = () => {
+    return (
+        <Container>
+            <div className='options'>
+                <div>
+                    <Tooltip 
+                        placement="bottom" 
+                        title={"Calender"}
+                        >
+                            <a href={`${urls.calender}`}>
+                                <FcCalendar />
+                            </a>
+                    </Tooltip> 
+                </div>
+                <div>
+                    <Tooltip 
+                        placement="bottom" 
+                        title={"Keep"}
+                        >
+                            <a href={`${urls.keep}`}>
+                                <FcIdea />
+                            </a>
+                    </Tooltip> 
+                </div>
+                <div>
+                    <Tooltip 
+                        placement="bottom" 
+                        title={"Tasks"}
+                        >
+                            <a href={`${urls.task}`}>
+                                <FcCheckmark />
+                            </a>
+                    </Tooltip> 
+                </div>
+                <div className='m-b'>
+                    <Tooltip 
+                        placement="bottom" 
+                        title={"Contact"}
+                        >
+                            <a href={`${urls.contact}`}>
+                                <FcManager />
+                            </a>
+                    </Tooltip> 
+                </div>
+                <div>
+                    <Tooltip 
+                        placement="bottom" 
+                        title={"Get add-ons"}
+                        >
+                            <a href={`${urls.plus}`}>
+                                <AiOutlinePlus />
+                            </a>
+                    </Tooltip> 
+                </div>
+            </div>
+        </Container>
```

Comment:
all code in this file follows a pattern, you have taken a `div` and then a `tooltip` and an `anchor` tag
we should convert it into an array `[{tooltipPlacement, tooltipTitle,  anchorIcon, anchorUrl}].map(item => component)` structure. adding a new item then would be straight forward changing in an array, code will also be more concise.